src/ZEO/zrpc/trigger.py

```diff
@@ -232,4 +232,4 @@ def _close(self):
             self.trigger.close()
 
         def _physical_pull(self):
-            self.trigger.send('x')
+            self.trigger.send(b'x')
```

Comment:
We are still supporting Python 3.2, which doesn't have the bytes literal prefix.  You can use something like the following:

```python
from ZODB._compat import ascii_bytes

TRIGGER_CHARACTER = ascii_bytes('x')
```

and then use that constant instead.